src/locales/de.json

```
@@ -173,5 +173,18 @@
   "setting.push.posts" : "reguläre Posts",
   "settings.push.thisDevice": "Dieses Gerät ({name})",
   "settings.title": "Einstellung",
+
+  "team.modal.editTitle.title": "",
+  "team.modal.editInfo.title": "",
+  "team.header.uploadButton": "",
+  "team.tabs.about": "",
+  "team.tabs.news": "",
+  "team.tabs.feed": "",
+  "team.description.edit": "",
+  "team.subteams.title": "",
```

thanks! here are the translations:
```suggestion
  "team.modal.editTitle.title": "Team Name",
  "team.modal.editInfo.title": "Team Infos",
  "team.header.uploadButton": "Hintergrund hochladen",
  "team.tabs.about": "Über",
  "team.tabs.news": "News",
  "team.tabs.feed": "Feed",
  "team.description.edit": "Bearbeiten",
  "team.subteams.title": "Unterteams",
```